cloudinit/util.py

```diff
@@ -857,7 +860,17 @@ def read_seeded(base="", ext="", timeout=5, retries=10, file_retries=0):
     if ud_resp.ok():
         ud = ud_resp.contents
 
-    return (md, ud)
+    vd = None
+    try:
+        vd_resp = url_helper.read_file_or_url(vd_url, timeout=timeout,
+                                              retries=retries)
+    except url_helper.UrlError:
```

Comment:
I think we should log here that there was an url error when reading the vendor-data